docs/standard/microservices-architecture/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -92,7 +92,7 @@ Let’s go back to the initial issue and its example. If the service crashes aft
 
 As mentioned earlier in the architecture section, you can have several approaches for dealing with this issue:
 
--   Using the full [Event Sourcing pattern](https://msdn.microsoft.com/en-us/library/dn589792.aspx).
+-   Using the full [Event Sourcing pattern](https://msdn.microsoft.com/library/dn589792.aspx).
```

Comment:
change this to https://docs.microsoft.com/azure/architecture/patterns/event-sourcing